### ISSUE TITLE
ValueError with datetime format

### ISSUE THREAD

**Issue by arosen93** [MasterScrat/Chatistics#3]
I am having the following issue with Pandas trying to read the datetime. Do you know why it's having trouble reading the full datetime? 

**Traceback**
```
105 Person Name (group? False )
Traceback (most recent call last):
  File "parse_messenger.py", line 71, in <module>
    timestamp = pd.to_datetime(content[:-7], format='%A, %B %d, %Y at %H:%M%p').strftime("%s")
  File "C:\Users\asros\Anaconda3\envs\py2\lib\site-packages\pandas\core\tools\datetimes.py", line 382, in to_datetime
    result = _convert_listlike(np.array([arg]), box, format)[0]
  File "C:\Users\asros\Anaconda3\envs\py2\lib\site-packages\pandas\core\tools\datetimes.py", line 306, in _convert_listlike
    raise e
ValueError: time data 'Monday, January 22, 2018 at 6:4' does not match format '%A, %B %d, %Y at %H:%M%p' (match)
```

**Text**:
```
</style><title>Conversation with Friend Name</title></head><body><a href="html/messages.htm">Back</a><br /><br /><div class="thread"><h3>Conversation with Friend Name</h3>Participants: Friend Name<div class="message"><div class="message_header"><span class="user">Friend Name</span><span class="meta">Monday, January 22, 2018 at 6:46pm CST</span></div></div><p>Oh wow</p><div class="message"><div class="message_header"><span class="user">My Name</span><span class="meta">Monday, January 22, 2018 at 6:44pm CST</span></div></div><p>blah blah blah message was here</p><div class="message"><div class="message_header"><span class="user">Friend Name</span><span class="meta">Monday, January 22, 2018 at 6:40pm CST</span></div></div><p>
```

**Comment by MasterScrat** [comment]
Oh yeah, that's because I didn't find how to parse the timezones properly so I hardcoded it for "UTC+02" and now it fails because you are in "CST".

I asked StackOverflow for help: https://stackoverflow.com/questions/48397197/parsing-dates-with-timezone-with-pandas

**Comment by MasterScrat** [comment]
Should be fixed with 95675c2, let me know if you still have problems.

**Comment by arosen93** [comment]
Thanks for addressing that so quickly! Looking forward to playing around with this utility.
